Modelica/Electrical/QuasiStatic/Machines/BasicMachines/package.mo

```diff
@@ -28,12 +29,17 @@ package BasicMachines "Basic machine models"
           fillPattern=FillPattern.Solid,
           points={{-70,-90},{-60,-90},{-30,-20},{20,-20},{50,-90},{60,-90},{
               60,-100},{-70,-100},{-70,-90}})}), Documentation(info="<html>
-This package contains components for modeling quasi-static electrical induction machines, based on space phasor theory:
+<p>This package contains components for modeling quasi-static electrical induction machines, based on space phasor theory:</p>
 <ul>
-<li>package InductionMachines: quasi-static models of three-phase induction machines</li>
-<li>package SynchronousMachines: quasi-static models of three-phase synchronous machines</li>
 <li>package Transformers: quasi-static three-phase transformers (see detailed documentation in subpackage)</li>
 <li>package Components: components for quasi-static modeling machines and transformers</li>
 </ul>
+<h4>Note</h4>
+<p>
+Quasi-static AC machines (induction machines and synchronous machines) based on space phasor therory are planned but not yet implemented. 
```

Comment:
We removed such promises in other locations. See #2334 or #2755. I'd recommend to not introduce it.

Reply:
resolved by addb4db8a..029293901